examples/k8s.yaml

```diff
@@ -9,24 +9,33 @@ metadata:
 data:
   otel-agent-config: |
     receivers:
-      opencensus: {}
+      otlp: {}
     exporters:
-      opencensus:
-        endpoint: "otel-collector.default:55678" # TODO: Update me
+      otlp:
+        endpoint: "otel-collector.default:55680" # TODO: Update me
     processors:
+      batch:
+      memory_limiter:
+        # Same as --mem-ballast-size-mib CLI argument
+        ballast_size_mib: 165
+        # 80% of maximum memory up to 2G
+        limit_mib: 400
+        # 25% of limit up to 2G
+        spike_limit_mib: 100
       queued_retry:
         num_workers: 4
         queue_size: 100
         retry_on_failure: true
     extensions:
+      health_check: {}
       zpages: {}
     service:
-      extensions: [zpages]
+      extensions: [health_check, zpages]
       pipelines:
         traces:
-          receivers: [opencensus]
-          processors: [queued_retry]
-          exporters: [opencensus]
+          receivers: [otlp]
+          processors: [batch, memory_limiter, queued_retry]
```

Comment:
Note the README currently recommends `memory_limiter` before `batch`: https://github.com/open-telemetry/opentelemetry-collector/blob/master/processor/README.md#recommended-processors

Reply:
good catch!